libraries/native/intrinsics.cpp

```diff
@@ -906,83 +906,77 @@ extern "C" {
-}
-
-int32_t alt_bn128_pair( const char* pairs, uint32_t pairs_len) {
-   return intrinsics::get().call<intrinsics::alt_bn128_pair>(pairs, pairs_len);
-}
-
-int32_t mod_exp( const char* base, uint32_t base_len, const char* exp, uint32_t exp_len, const char* mod, uint32_t mod_len, char* result, uint32_t result_len) {
-   return intrinsics::get().call<intrinsics::mod_exp>(base, base_len, exp, exp_len, mod, mod_len, result, result_len);
-}
-
-void sha3( const char* data, uint32_t data_len, char* hash, uint32_t hash_len, int32_t keccak ) {
-   intrinsics::get().call<intrinsics::sha3>(data, data_len, hash, hash_len, keccak);
-}
-
-int32_t bls_g1_add(const char* op1, uint32_t op1_len, const char* op2, uint32_t op2_len, char* res, uint32_t res_len)
-{
-    return intrinsics::get().call<intrinsics::bls_g1_add>(op1, op1_len, op2, op2_len, res, res_len);
-}
-
-int32_t bls_g2_add(const char* op1, uint32_t op1_len, const char* op2, uint32_t op2_len, char* res, uint32_t res_len)
-{
-    return intrinsics::get().call<intrinsics::bls_g2_add>(op1, op1_len, op2, op2_len, res, res_len);
-}
-
-int32_t bls_g1_mul(const char* point, uint32_t point_len, const char* scalar, uint32_t scalar_len, char* res, uint32_t res_len)
-{
-    return intrinsics::get().call<intrinsics::bls_g1_mul>(point, point_len, scalar, scalar_len, res, res_len);
-}
-
-int32_t bls_g2_mul(const char* point, uint32_t point_len, const char* scalar, uint32_t scalar_len, char* res, uint32_t res_len)
-{
-    return intrinsics::get().call<intrinsics::bls_g2_mul>(point, point_len, scalar, scalar_len, res, res_len);
-}
-
-int32_t bls_g1_exp(const char* points, uint32_t points_len, const char* scalars, uint32_t scalars_len, uint32_t n, char* res, uint32_t res_len)
-{
-    return intrinsics::get().call<intrinsics::bls_g1_exp>(points, points_len, scalars, scalars_len, n, res, res_len);
-}
-
-int32_t bls_g2_exp(const char* points, uint32_t points_len, const char* scalars, uint32_t scalars_len, uint32_t n, char* res, uint32_t res_len)
-{
-    return intrinsics::get().call<intrinsics::bls_g2_exp>(points, points_len, scalars, scalars_len, n, res, res_len);
-}
-
-int32_t bls_pairing(const char* g1_points, uint32_t g1_points_len, const char* g2_points, uint32_t g2_points_len, uint32_t n, char* res, uint32_t res_len)
-{
-    return intrinsics::get().call<intrinsics::bls_pairing>(g1_points, g1_points_len, g1_points, g1_points_len, n, res, res_len);
-}
-
-int32_t bls_g1_map(const char* e, uint32_t e_len, char* res, uint32_t res_len)
-{
-    return intrinsics::get().call<intrinsics::bls_g1_map>(e, e_len, res, res_len);
-}
-
-int32_t bls_g2_map(const char* e, uint32_t e_len, char* res, uint32_t res_len)
-{
-    return intrinsics::get().call<intrinsics::bls_g2_map>(e, e_len, res, res_len);
-}
-
-int32_t bls_fp_mod(const char* s, uint32_t s_len, char* res, uint32_t res_len)
-{
-    return intrinsics::get().call<intrinsics::bls_fp_mod>(s, s_len, res, res_len);
-}
```

Comment:
I don't think you should change the indentation here.

Reply:
I just aligned it with the rest of file. generally I'm not hard on this, what was the latest on clang format? Just live with people's individual styles?